tests/baselines/reference/jsxChildrenArrayWrongType.errors.txt

```diff
@@ -20,9 +20,9 @@ index.tsx(11,6): error TS2769: No overload matches this call.
          ~~~
 !!! error TS2769: No overload matches this call.
 !!! error TS2769:   Overload 2 of 2, '(props: PropsType, context: any): Foo', gave the following error.
-!!! error TS2769:     Type 'unknown' is not assignable to type 'string | boolean'.
+!!! error TS2769:     Type 'unknown' is not assignable to type 'string'.

```

Comment:
This does seem worse overall, but it feels like JSX children should have better messages.

Reply:
I touched on this [here](https://github.com/microsoft/TypeScript/pull/57537#discussion_r1502163633). Let me know if that makes any sense or if you think I should work more on improving this case.